common-lib/src/cmr/common/api/context.clj

```diff
@@ -48,3 +51,12 @@
   (when-let [request-id (context->request-id context)]
     {REQUEST_ID_HEADER request-id}))
 
+(defn context->user-id
```

Comment:
It's good that you are doing this. I almost did it myself, but decided to shelf it as I am not sure where to put this function. I have thought about putting it in `cmr.search.api.request-context-user-augmenter` as that is most closely related to the `util/lazy-assoc` and this function is only used by search-app. I would like to hear what other people think about it.